docs/devguide.md

```diff
@@ -343,6 +343,10 @@ Once the Pull Request has been created, it will automatically be built
 and the tests run. The unit and integration tests will run in travis,
 and Jenkins will run the e2e tests.
 
+You can use the [Prow /cc command](https://prow.k8s.io/command-help#cc)
```

Comment:
Are there any other Prow commands that work? I tried to use /retest just a few minutes ago. 😀

Reply:
there's a config file that I can't find

Reply:
https://github.com/kubernetes/test-infra/blob/50106cd3395a282d648800e87dc85409b1aca70a/prow/plugins.yaml#L352-L358
any commands supported by those plugins, so more research required.

Reply:
Slick, thanks!

Reply:
Maybe in a follow-up we can provide a link to the prow doc for each of the supported commands?

Reply:
follow-up please. I am looking into more prow details today for a little while.